### PR TITLE
Add service to set IK output state

### PR DESCRIPTION
**Pull request by dogoepp**
Follows the discussion in #20.

### CODE REVIEW

**Comment by dogoepp** [comment]
If you think it's a sensible choice, I could also make it possible for the user to choose between services or topics, for instance through a parameter.

File: phidgets_ik/src/ik_ros_i.cpp [hunk 1]
```diff
@@ -49,6 +49,7 @@ void IKRosI::initDevice()
     snprintf(topicname, sizeof(topicname), "DigitalOutput%02d", i);
     boost::shared_ptr<OutputSetter> s (new OutputSetter(ik_handle_, i));
     s->subscription = nh_.subscribe(topicname, 1, &OutputSetter::set_msg_callback, s);
+    s->service = nh_.advertiseService(topicname, &OutputSetter::set_srv_callback, s);
```
Comment (mintar) [inline phidgets_ik/src/ik_ros_i.cpp]: The services should be called "set_digital_output00", not "DigitalOutput00". Also, (although it's a bit unrelated to this PR, but it should be changed nonetheless), the other topics should be called "digital_output00", "digital_input00", analog_input00".

**Comment by mintar** [comment]
Other than the change I've suggested above, this looks good to me!

**Comment by mintar** [comment]
I wouldn't add a parameter to choose between topics and services. The user can just use either and ignore the other.

**Comment by dogoepp** [comment]
All right. I changed the names for the topics and services.

**Comment by mintar** [comment]
Many thanks!

**Comment by dogoepp** [comment]
Thank you for accepting these changes !

**Pull request merged**